example/centos/centos-9-aarch64-vhd.yaml

```diff
@@ -0,0 +1,315 @@
+            - patch
+            - rng-tools
+            - selinux-policy-targeted
+            - shim-aa64
+            - uuid
+            - xfsprogs
+            - yum-utils
+          exclude:
+            - NetworkManager-config-server
+            - aic94xx-firmware
+            - alsa-firmware
+            - alsa-lib
+            - alsa-sof-firmware
+            - alsa-tools-firmware
+            - biosdevname
+            - bolt
+            - buildah
+            - cockpit-podman
+            - containernetworking-plugins
+            - dnf-plugin-spacewalk
+            - dracut-config-rescue
+            - glibc-all-langpacks
+            - iprutils
+            - ivtv-firmware
+            - iwl100-firmware
+            - iwl1000-firmware
+            - iwl105-firmware
+            - iwl135-firmware
+            - iwl2000-firmware
+            - iwl2030-firmware
+            - iwl3160-firmware
+            - iwl3945-firmware
+            - iwl4965-firmware
+            - iwl5000-firmware
+            - iwl5150-firmware
+            - iwl6000-firmware
+            - iwl6000g2a-firmware
+            - iwl6000g2b-firmware
+            - iwl6050-firmware
+            - iwl7260-firmware
+            - libertas-sd8686-firmware
+            - libertas-sd8787-firmware
+            - libertas-usb8388-firmware
+            - plymouth
+            - podman
+            - python3-dnf-plugin-spacewalk
+            - python3-hwdata
+            - python3-rhnlib
+            - rhn-check
+            - rhn-client-tools
+            - rhn-setup
+            - rhnlib
+            - rhnsd
+            - usb_modeswitch
+  kernel:
+    cmdline: ro loglevel=3 console=tty1 console=ttyS0 earlyprintk=ttyS0 rootdelay=300
+    package:
+      otk.external.osbuild-get-dnf4-package-info:
+        packageset: ${packages.os}
+        packagename: "kernel"
+
+otk.include: "common/partition-table/aarch64/vhd.yaml"
+
+otk.target.osbuild:
+  pipelines:
+    - otk.include: "pipeline/build/generic.yaml"
+    - name: os
+      build: name:build
+      stages:
+        - type: org.osbuild.kernel-cmdline
+          options:
+            root_fs_uuid: ${filesystem.const.partition_map.root.uuid}
+            kernel_opts: ${kernel.cmdline}
+        - otk.external.osbuild-make-depsolve-dnf4-rpm-stage:
+            packageset: ${packages.os}
+            gpgkeys:
+              otk.include: "common/gpgkeys.yaml"
+        - type: org.osbuild.fix-bls
+          options:
+            prefix: ''
+        - type: org.osbuild.locale
+          options:
+            language: en_US.UTF-8
+        - type: org.osbuild.keymap
+          options:
+            keymap: us
+            x11-keymap:
+              layouts:
+                - us
+        - type: org.osbuild.timezone
+          options:
+            zone: Etc/UTC
+        - type: org.osbuild.sysconfig
+          options:
+            kernel:
+              update_default: true
+              default_kernel: kernel-core
```

Comment:
(for later) this is a curious one, qcow2/ami uses "kernel" here, would be nice to know why vhd uses kernel-core